src/main/java/uk/gov/ons/census/notifyprocessor/service/FulfilmentRequestService.java

```diff
@@ -84,7 +90,8 @@ public void processMessage(ResponseManagementEvent fulfilmentEvent) {
     rabbitTemplate.convertAndSend(enrichedFulfilmentExchange, "", enrichedFulfilmentRequest);
   }
 
-  private UacQid getUacQidPair(int questionnaireType, UUID caseId) {
+  private UacQid getUacQidPair(
+      int questionnaireType, UUID caseId, UUID transactionId, String source, String channel) {
```

Comment:
I'd have just passed the source `Event` but this will work. Don't go pass more than 5 parameters to a function, is a good rule, which this meets.